database/cnl.sql

```diff
@@ -1544,6 +1544,8 @@ CREATE TABLE IF NOT EXISTS quick_packs (
     customer_id CHAR(36) NOT NULL,
     name VARCHAR(255) NOT NULL, 
     description TEXT,
+    active ENUM('Y', 'N'),
+    lot_qty INT
```

Comment:
comma(,) missing here
Also change the description from TEXT datatype to VARCHAR(1024)

Reply:
Updated Thulasi.